app/src/main/java/com/asfoundation/wallet/billing/analytics/WalletsAnalytics.kt

```diff
@@ -0,0 +1,143 @@
+package com.asfoundation.wallet.billing.analytics
+
+import cm.aptoide.analytics.AnalyticsManager
+import java.util.*
+
+class WalletsAnalytics(private val analytics: AnalyticsManager) : WalletsEventSender {
+
+  override fun sendCreateBackupEvent(action: String, context: String,
+                                     status: String) {
+    sendCreateBackupEvent(action, context, status, null)
+  }
+
+  override fun sendCreateBackupEvent(action: String, context: String,
+                                     status: String,
+                                     errorDetails: String?) {
```

Comment:
You can use default value = null, to avoid passing null on the methods that call this